avocado/utils/multipath.py

```diff
@@ -58,6 +58,7 @@ def form_conf_mpath_file(blacklist="", defaults_extra=""):
     # multipath.conf file to take effect.
     time.sleep(5)
     service.SpecificServiceManager(get_svc_name()).restart()
+    time.sleep(4)
```

Comment:
@apahim we need the sleep() at both the places, first one is for multipath.conf file( as it is commented)and second sleep() is for multipath services.